render/render.js

```diff
@@ -51,18 +51,17 @@ module.exports = function($window) {
 			vnode.state = {}
 			if (vnode.attrs != null) initLifecycle(vnode.attrs, vnode, hooks)
 			switch (tag) {
-				case "#": return createText(parent, vnode, nextSibling)
```

Comment:
I don't have a strong opinion one way or another, but why'd this change?

Reply:
The individual `createX` methods used to return a DOM node that was used in `createComponent()` and `updateNodes()` (in the map-based diff section).

This isn't needed anymore (and wasn't necessary in the first place AFAICT,  since `insertNode()` in `createComponent()` was redundant and we can get `v.dom` in `updateNodes()`).

So I made it explicit that `createNode()` wasn't returning anything.